code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -28,10 +28,10 @@ const CourseProgress = React.createClass({
 
   render() {
     const groups = _.groupBy(this.props.stages, stage => (stage.flex_category || 'Content'));
-
     let count = 1;
     const rows = _.map(groups, (stages, group) =>
       <div key={group}>
+        <a name={group.toLowerCase().replace(' ', '-')}/>
```

Comment:
Yeah, the `name` attribute for anchor tags was [deprecated in XHTML 1.0](https://www.w3.org/TR/xhtml1/#C%5F8)